pulse2percept/oyster.py

```diff
@@ -108,10 +108,7 @@ def jansonius(nCells, nR, center=np.array([15, 2]),
     return x, y
 
 
-def makeAxonMap(xg, yg, axon_lambda=1, nCells=500, nR=801, min_weight=.001,
-                center=np.array([15, 2]),
-                rot=0 * np.pi / 180, scale=1, bs=-1.9, bi=.5, r0=4,
-                maxR=45, angRange=60):
+def makeAxonMap(xg, yg, jan_x, jan_y, axon_lambda=1, min_weight=.001):
```

Comment:
The goal here was to give a user access to the axon map (`jan_x` and `jan_y`) for visual investigation and pretty-plotting, which is the output of `jansonius`. In the old code, the `Retina` object called `makeAxonMap`, which called `jansionius` under the hood, therefore not exposing `jan_x` and `jan_y` to theuser. Now `Retina` calls both these functions, and stores `jan_x` and `jan_y` in the .npz file.

Funny enough, `nCells` and `nR` were never available to the `Retina` API, so I hardcoded them. I don't think they are supposed to change, but I can always bring them back if we need them.